### ISSUE TITLE
Precompiled 2.1.1 not working on CentOS 5.6

### ISSUE THREAD

**Issue by kmart1234** [ariya/phantomjs#14099]
Got this working on several CentOS 6 servers, but not working on CentOS 5. Does not appear to be self-contained.

[root@server ~]# phantomjs -v
phantomjs: /lib64/libz.so.1: no version information available (required by phantomjs)
phantomjs: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.9' not found (required by phantomjs)
phantomjs: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.11' not found (required by phantomjs)
phantomjs: /lib64/libc.so.6: version `GLIBC_2.9' not found (required by phantomjs)
phantomjs: /lib64/libc.so.6: version `GLIBC_2.10' not found (required by phantomjs)
phantomjs: /lib64/libc.so.6: version `GLIBC_2.7' not found (required by phantomjs)
[root@server ~]#

Using glibc-2.5-123.el5_11.1

md5sum of unpacked binary that was downloaded from phantomjs.org
3f4bbbe5acd45494d8e52941936235f2  phantomjs

**Comment by AJRobson1** [comment]
I had this same problem at work but with 5.9. I tried compiling it myself and even using wine to run the Windows executable. 

Those libraries don't come with that version of CentOS, if you google phantomJS CentOS  5.9 you will find a guide to take the missing libraries from a new version and add them to your version but it's a bit of hack and might cause more problems. 

In the end I just convinced the build team to upgrade to the latest version of CentOS and it worked perfectly, 5.6 is five years old.
